src/controllers/link.js

```diff
@@ -0,0 +1,13 @@
+import Link from '../models/UrlShorten';
+module.exports = {
+  redirectLink: async (req, res)=>{
+    const {id} = req.params;
+    const fondLink = await Link.findOne({urlCode: id})
+    if(fondLink){
+      const actualLocation = foundLink.long_url
```

Comment:
I guess you meant fondLink.long_url  in line 7

Reply:
Ah yes. I will look into that. Thanks

Reply:
Did you test this before pushing?

Reply:
It will be fixed in a bit

Reply:
Hi @rexdavinci, has it been fixed?